packages/fleather/lib/src/widgets/field.dart

```diff
@@ -91,6 +91,12 @@ class FleatherField extends StatefulWidget {
   /// the text field from the clipboard.
   final bool enableInteractiveSelection;
 
+  /// Defines how to measure the width of the rendered when [readOnly] is `true`
```

Comment:
I think it was meant to be "Defines how to measure the width of the rendered text when [readOnly] is true. ...", wasn't it?